samples/app-anonymous-users/nodejs/ClientApp/src/components/share-to-meeting.jsx

```diff
@@ -0,0 +1,18 @@
+// <copyright file="share-to-meeting.jsx" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// Default browser page from where content can directly shared to meeting.
+// </copyright>
+
+const ShareToMeeting = props => {
+
+    return (
+        <>
+		    <div class="surface">
+                <h3 id="tabheader">Welcome to the app that support anonymous user support in teams meeting apps.</h3>
```

Comment:
Welcome to the app that support anonymous user support in `Teams` meeting apps.

Reply:
Fixed